src/tests/Calculator.test.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react';
+import Calculator from '../components/Calculator';
+
+describe('render the calculator', () => {
+  test('displays the calculator text', () => {
+    const { container } = render(<Calculator />);
+    const textCalculator = screen.getByText(/Lets doing some math!/i);
+    expect(textCalculator).toBeInTheDocument();
```

Comment:
- [ ] You did great work team 👍  , however, all tests are not passing. currently, I tried to change this line:
````js
const textCalculator = screen.getByText(/Lets doing some math!/i);
````
to 
````js
const textCalculator = screen.getByText(/Lets do some math!/i);
````
now it is working as we are testing the component using Jest snapshots. Kindly fix it 😄 

 
before
![Screenshot 2023-08-16 160238](https://github.com/fatima-najafi/math-magicians/assets/57408419/b96235a6-e2d9-4a11-bf00-f1e6748e4558)
after

![Screenshot 2023-08-16 160150](https://github.com/fatima-najafi/math-magicians/assets/57408419/0ac1a678-8400-4416-9a78-e082857d1c9d)